tests/scenarios/compat-stage2-test.ts

```diff
@@ -40,6 +40,9 @@ stage2Scenarios
         'check-resolution-target.js': 'export {}',
       },
     });
+
+    (depC.pkg['ember-addon'] as any).before = ['dep-a', 'dep-b', 'in-repo-d', 'in-repo-a'];
```

Comment:
Why do we have this?

Reply:
Using windows real path instead of short path alias caused this